packages/loopring_v3/circuit/Utils/Data.h

```diff
@@ -14,14 +16,130 @@ using json = nlohmann::json;
 namespace Loopring
 {
 
-enum class BlockType
-{
-    RingSettlement = 0,
+static auto dummySpotTrade = R"({
+    "fFillS_A": 0,
+    "fFillS_B": 0,
+    "orderA": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "tokenS": 0,
+        "tokenB": 1,
+        "validSince": 0,
+        "validUntil": 4294967295
+    },
+    "orderB": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "reduceOnly": 0,
+        "tokenS": 1,
+        "tokenB": 0,
+        "validSince": 0,
+        "validUntil": 4294967295
+    }
+})"_json;
+
+static auto dummyTransfer = R"({
+    "fromAccountID": 0,
+    "toAccountID": 2,
+    "amount": "0",
+    "fee": "0",
+    "feeTokenID": 0,
+    "tokenID": 0,
+    "validUntil": 4294967295,
+    "type": 0,
+    "ownerFrom": "0",
+    "to": "2",
+    "dualAuthorX": "0",
+    "dualAuthorY": "0",
+    "data": "0",
+    "payerToAccountID": 2,
+    "payerTo": "2",
+    "payeeToAccountID": 2,
+    "nonce": 0
+})"_json;
+
+static auto dummyWithdraw = R"({
+    "owner": "0",
+    "accountID": 0,
+    "tokenID": 0,
+    "amount": "0",
+    "feeTokenID": 0,
+    "fee": "0",
+    "to": "0",
+    "dataHash": "0",
+    "minGas": 0,
+    "type": 0
+})"_json;
+
+static auto dummyAccountUpdate = R"({
+    "owner": "0",
+    "accountID": 0,
+    "nonce": 0,
+    "publicKeyX": "13060336632196495412858530687189935300033555341384637843571668213752389743866",
```

Comment:
is this a random public key, do we need the private key?

Reply:
No, it's just a valid public key so the circuits accepts it, but it will never end up being used.

Reply:
To make an auditor not wonder, maybe we should say this randomly selected pubkey is generated with a pre-defined method, for example: using xxx.com, with seed `Loopring3.6(zkRollup)`

Reply:
It's a valid point on the baby jubjub curve, which is easily verified by filling in the points in the curve function. How it's generated isn't really important and I don't think it's especially helpful because it will be harder to validate how the point is generated than just that the point is valid.

We could also just use one of the points here I guess: https://github.com/ethereum/EIPs/blob/41569d75e42da2046cb18fdca79609e18968af47/eip-draft_babyjubjub.md